rocq-runtime.opam.template

```diff
@@ -20,3 +20,7 @@ build: [
   ]
   ["dune" "install" "-p" name "--create-install-files" name]
 ]
+conflicts: [
+  "coq" { < "8.17" }
+  "coq-core" { < "8.21" }
+]
```

Comment:
IIUC this could be put in dune-project instead of the template

Reply:
Indeed, done